Add net-istio webhook to spec.registry.override (#3821)
knative/docs: docs/admin/install/operator/configuring-serving-cr.md

      webhook: docker.io/knative-images-repo4/webhook:v0.13.0
      autoscaler-hpa: docker.io/knative-images-repo5/autoscaler-hpa:v0.13.0
      networking-istio: docker.io/knative-images-repo6/prefix-networking-istio:v0.13.0
      net-istio/webhook: docker.io/knative-images-repo6/networking-istio-webhook:v0.13.0
      queue-proxy: docker.io/knative-images-repo7/queue-proxy-suffix:v0.13.0
```

> !!! note
>     If the container name is not unique across all of the deployments, daemonsets and jobs,
>     you can prefix the container name with the "parent's" name and a slash, e.g. `net-istio/webhook`.

Reviewer: ```suggestion
!!! note
    If the container name is not unique across all Deployments, DaemonSets and Jobs,
    you can prefix the container name with the parent container name and a slash. For example, `net-istio/webhook`.
```

Author: Thank you! Merged it.
